### PR TITLE
Add Svelte-Spectre UI kit

### PR DESCRIPTION
**Pull request by blokhin**
https://github.com/basf/svelte-spectre

### CODE REVIEW

**Comment by jaywcjlove** [comment]
There is no official documentation. @blokhin

**Comment by blokhin** [comment]
Sorry, forgot to add a [link](https://svelte-spectre.surge.sh) to the repo readme. Done.

File: README.md [hunk 1]
```diff
@@ -440,6 +440,7 @@ _Support [React](https://github.com/facebook/react), [Angular](https://github.co
 - [spectre](https://picturepan2.github.io/spectre/) <img align="bottom" height="13" src="https://img.shields.io/github/stars/picturepan2/spectre.svg?label=" /> - A Lightweight, Responsive and Modern CSS Framework. [![Open-Source Software][OSS Icon]](https://github.com/picturepan2/spectre)
 - [SUI](http://m.sui.taobao.org/) <img align="bottom" height="13" src="https://img.shields.io/github/stars/sdc-alibaba/SUI-Mobile.svg?label=" /> - A framework based on the development of Framework7 UI library. [![Open-Source Software][OSS Icon]](https://github.com/sdc-alibaba/SUI-Mobile)
 - [SuperUI](http://www.supermgr.cn/) <img align="bottom" height="13" src="https://img.shields.io/github/stars/tzhsweet/superui.svg?label=" /> - 让体验变得如此完美，开发变得如此简单。 [![Open-Source Software][OSS Icon]](https://github.com/tzhsweet/superui)
+- [svelte-spectre](https://github.com/basf/svelte-spectre) <img align="bottom" height="13" src="https://img.shields.io/github/stars/basf/svelte-spectre.svg?label=" /> - A Spectre kit for Svelte framework. [![Open-Source Software][OSS Icon]](https://github.com/basf/svelte-spectre)
```
Comment (jaywcjlove) [inline README.md:443]: @blokhin Need to add the official website address, like this:

```md
- [svelte-spectre](https://svelte-spectre.surge.sh/) <img align="bottom" height="13" src="https://img.shields.io/github/stars/basf/svelte-spectre.svg?label=" /> - A Spectre kit for Svelte framework. [![Open-Source Software][OSS Icon]](https://github.com/basf/svelte-spectre)
```

**Comment by blokhin** [comment]
@jaywcjlove sorry for such a long delay with the edits...

**Pull request merged**